src/plugins/intel_gpu/src/graph/layout_optimizer.cpp

```diff
@@ -1597,6 +1598,13 @@ format layout_optimizer::get_preferred_format(program_node& node) {
                     expected = expected_conv_fmt;
             }
         }
+
+        // In case of input -> ... -> quantize -> concat
+        if (expected == format::any
+            && (node.get_users().size() == 1 && node.get_users().front()->is_type<concatenation>())
+            && (layout.batch() < 4 && layout.feature() < 4)) {
+                expected = format::get_default_format(layout.get_rank(), false, false);
+        }
```

Comment:
How do you think about adding a test case for this pattern?

Reply:
I've found only one model has this pattern. So, I think it is not necessary.

Reply:
I've added a test case for this.